mungegithub/mungers/matchers/event/event.go

```diff
@@ -0,0 +1,71 @@
+package event
+
+import (
+	"time"
+
+	"github.com/google/go-github/github"
+)
+
+type Matcher interface {
+	Match(event *github.IssueEvent) bool
+}
+
+type And []Matcher
```

Comment:
suggest move and/or/not to separate file so that the others are obvious and self-documenting.